lib/getConfig.js

```diff
@@ -10,17 +11,51 @@ const getConfig = (altPath) => {
   const configObject = json.readToObjSync(pathString);
   const globalConfig = json.readToObjSync(path.join(homedir, '.sgcrc'));
   const packageConfig = json.readToObjSync(path.join(cwd, 'package.json')).sgc;
-  const sgcrcDefaultConfig = json.readToObjSync(path.join(__dirname, '..', '.sgcrc_default'));
+  const sgcrcDefaultConfig = json.readToObjSync(path.join(__dirname, '..', '.sgcrc'));
+  const sgcrcTestDefaultConfig = json.readToObjSync(path.join(__dirname, '..', '.sgcrc_default'));
+
+  const sgcrcDefault = sgcrcDefaultConfig || sgcrcTestDefaultConfig;
 
   // priority order (1. highest priority):
   // 1. local config
   //   - 1. .sgcrc
   //   - 2. (package.json).sgc
   // 2. global config
-  // 3. default config from ../.sgcrc_default
-  const config = configObject || packageConfig || globalConfig || sgcrcDefaultConfig;
+  // 3. default config
+  //   - 1. from ../.sgcrc
+  //   - 2. test case ../.sgcrc is renamed to ../.sgcrc_default
+  let config = configObject || packageConfig || globalConfig || sgcrcDefault;
+
+  config.inherit = config.inherit === undefined ? {} : config.inherit;
+
+  // set defaults
+  let configDefaults = {
```

Comment:
Hm, yes that's good. Those defaults are just necessary where it is set to `true`